samples/snippets/participant_management.py

```diff
@@ -0,0 +1,188 @@
+    response = client.analyze_content(participant=participant_path,
+                                      text_input=text_input)
+    print('AnalyzeContent Response:')
+    print('Reply Text: {}'.format(response.reply_text))
+
+    for suggestion_result in response.human_agent_suggestion_results:
+        if suggestion_result.error is None:
+            print('Error: {}'.format(suggestion_result.error.message))
+        if suggestion_result.suggest_articles_response:
+            for answer in suggestion_result.suggest_articles_response.article_answers:
+                print('Article Suggestion Answer: {}'.format(answer.title))
+                print('Answer Record: {}'.format(answer.answer_record))
+        if suggestion_result.suggest_faq_answers_response:
+            for answer in suggestion_result.suggest_faq_answers_response.faq_answers:
+                print('Faq Answer: {}'.format(answer.answer))
+                print('Answer Record: {}'.format(answer.answer_record))
+        if suggestion_result.suggest_smart_replies_response:
+            for answer in suggestion_result.suggest_smart_replies_response.smart_reply_answers:
+                print('Smart Reply: {}'.format(answer.reply))
+                print('Answer Record: {}'.format(answer.answer_record))
+
+    for suggestion_result in response.end_user_suggestion_results:
+        if suggestion_result.error:
+            print('Error: {}'.format(suggestion_result.error.message))
+        if suggestion_result.suggest_articles_response:
+            for answer in suggestion_result.suggest_articles_response.article_answers:
+                print('Article Suggestion Answer: {}'.format(answer.title))
+                print('Answer Record: {}'.format(answer.answer_record))
+        if suggestion_result.suggest_faq_answers_response:
+            for answer in suggestion_result.suggest_faq_answers_response.faq_answers:
+                print('Faq Answer: {}'.format(answer.answer))
+                print('Answer Record: {}'.format(answer.answer_record))
+        if suggestion_result.suggest_smart_replies_response:
+            for answer in suggestion_result.suggest_smart_replies_response.smart_reply_answers:
+                print('Smart Reply: {}'.format(answer.reply))
+                print('Answer Record: {}'.format(answer.answer_record))
+
+    return response
+
+
+# [END dialogflow_analyze_content_text]
+
+
+# [START dialogflow_streaming_analyze_content_audio]
+def streaming_analyze_content_audio(participant_name,
+                                    sample_rate_herz,
+                                    stream,
+                                    timeout,
+                                    enable_extended_streaming=False):
+    """Stream audio to Dialogflow and receive transcripts and suggestions.
+
+    Args:
+        participant_name: resource name of the participant.
+        sample_rate_herz: herz rate of the sample.
+        audio_generator: a sequence of audio data.
```

Comment:
This doesn't match the argument list above.

Reply:
Fixed.